plugins/tekton/README.md

```diff
@@ -172,4 +172,4 @@ Tekton is a front-end plugin that enables you to view the `PipelineRun` resource
 
    The **TEKTON** tab contains the list of pipeline runs related to a cluster. The list contains pipeline run details, such as **NAME**, **STATUS**, **TASK STATUS**, **STARTED**, and **DURATION**.
 
-   ![tekton-tab](./images/tekton-plugin-user2.png)
+    ![tekton-tab](./images/tekton-plugin-user2.png)
```

Comment:
job is failing https://github.com/janus-idp/backstage-plugins/actions/runs/5877416693/job/15937509561?pr=606 due to formatting issue from prettier for this

Reply:
Done

Reply:
The job is still failing